teamcity/manifest.json

```diff
@@ -37,6 +37,11 @@
       "configuration": {
         "spec": "assets/configuration/spec.yaml"
       },
+      "metrics": {
```

Comment:
nit - A while ago, we removed the `Integration` word from all the integrations titles. Not sure why this one still have. Could you also update the title?